build.gradle

```diff
@@ -23,6 +23,12 @@ repositories {
 	// for more information about repositories.
 	maven { url = 'https://maven.terraformersmc.com/releases/'}
 	maven { url = 'https://maven.shedaniel.me/'}
+	maven { url = 'https://maven.pkg.github.com/gniftygnome/maybe-data'
+		credentials {
```

Comment:
Maybe data is actually available on jitpack; although you have use the latest commit, because no jar was uploaded to github on releease

Reply:
Oh, I wondered where you were getting it.  I've committed the configuration change to use jitpack.